### PR TITLE
Integrate Styled components in Contact and Terms Page

### LINKED ISSUE #2350: Integrate Styled components in Contact and Terms Page
**Actual Behaviour**
Not present.

**Expected Behaviour**
Integrate Styled components. 
Change px to rem.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2350 
Changes: 
Integrate Styled components in Contact and Terms Page
change px to rem
Demo Link: https://pr-2365-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]
Screenshot 2019-06-24 at 9 36 13 AM
Screenshot 2019-06-24 at 9 37 07 AM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 Fixed

File: src/components/Contact/Contact.react.js [hunk 1]
```diff
@@ -1,8 +1,53 @@
...
+    width: 80%;
+  }
+`;
+
+const ContactContent = styled.div`
+  ${commonContactContent};
+  border-bottom: 0.063rem solid #eee;
+`;
+
+const ContactContentLast = styled.div`
```
Comment (fragm3) [inline src/components/Contact/Contact.react.js]: Use `last-child` selector for achieving this

File: src/components/Contact/Contact.react.js [hunk 2]
```diff
@@ -1,8 +1,53 @@
...
+  ${commonContactContent};
+`;
+
+const H5 = styled.h5`
+  font-size: 1.125rem;
+  font-weight: 400;
+`;
+
+const Para = styled.p`
```
Comment (fragm3) [inline src/components/Contact/Contact.react.js]: Rename it `Text`

**Comment by harsh-jindal** [comment]
@fragm3 Done

**Comment by harsh-jindal** [comment]
@fragm3 Fixed.

**Pull request merged**